mavsdk/system.py

```diff
@@ -352,21 +352,24 @@ def _start_mavsdk_server(system_address, port, sysid, compid):
             from importlib_resources import path
 
         try:
-            with path(bin, 'mavsdk_server') as backend:
+            mavsdk_exec_name = "mavsdk_server"
+            if sys.platform.startswith('win'):
+                mavsdk_exec_name = "mavsdk_server.exe"
+            with path(bin, mavsdk_exec_name) as backend:
                 bin_path_and_args = [os.fspath(backend),
-                                     "-p", str(port),
-                                     "--sysid", str(sysid),
-                                     "--compid", str(compid)]
-                if system_address:
-                    bin_path_and_args.append(system_address)
-                p = subprocess.Popen(bin_path_and_args,
-                                     shell=False,
-                                     stdout=subprocess.PIPE,
-                                     stderr=subprocess.STDOUT)
-
-                logger = logging.getLogger(__name__)
-                log_thread = _LoggingThread(p.stdout, logger.debug)
-                log_thread.start()
+                                    "-p", str(port),
+                                    "--sysid", str(sysid),
+                                    "--compid", str(compid)]
+            if system_address:
+                bin_path_and_args.append(system_address)
+            p = subprocess.Popen(bin_path_and_args,
+                                    shell=False,
+                                    stdout=subprocess.PIPE,
+                                    stderr=subprocess.STDOUT)
+
+            logger = logging.getLogger(__name__)
+            log_thread = _LoggingThread(p.stdout, logger.debug)
+            log_thread.start()
```

Comment:
hmm are you entirely sure that changing the indentation here does not change the behavior?

I'd rather keep the old indentation that I know works, if there is not a good reason to change it :blush: